github-management/subproject-site-requests.md

```diff
@@ -85,7 +85,7 @@ To do so, create an issue using the [DNS Update Request] issue template in the
   # https://github.com/kubernetes-sigs/<project> (<contact or owners>)
   <project>.sigs:
     type: CNAME
-    value: <netlify url>
+    value: kubernetes-docs.netlifyglobalcdn.com.
```

Comment:
IIRC that is for kubernetes.io only. I believe that is the only one that is setup with their high performance CDN.

For other sites the docs  reference pointing the cname to `<name>.netlify.app.`

https://docs.netlify.com/domains-https/custom-domains/configure-external-dns/